DESCRIPTION

```diff
@@ -9,11 +9,14 @@ Description: More about what it does (maybe more than one line)
 License: BSD_2_clause + file LICENSE
 Encoding: UTF-8
 LazyData: true
+Depends:
+    R (>= 2.10)
 Imports:
-    curl,
     V8,
     jsonlite,
```

Comment:
Let's move these two packages to `Suggests`.  

They are not required to get at the data, which is the main goal of the package.